.github/workflows/release.yml

```diff
@@ -15,14 +15,20 @@ jobs:
     runs-on: ubuntu-latest
     name: "Release a new version"
     steps:
+      - name: Get auth token
+        id: token
+        uses: actions/create-github-app-token@3378cda945da322a8db4b193e19d46352ebe2de5 #v.1.10.4
```

Comment:
hmm the tag here doesn't look right

Reply:
I'm using the one from maintained by GitHub https://github.com/actions/create-github-app-token

Reply:
yes but `v.1.10.4` looks like a mistake